### DISCUSSION TITLE
Import/export templates

### DISCUSSION THREAD

**Discussion by xahodo** [vkbo/novelWriter#1886] [Ideas]
It would be useful if we (the users) could share templates among one another. Or at least import templates from somewhere. It doesn't by all necessity need to be from somewhere on the web (although this would be fantastic), it could be as simple as importing/exporting a selected file/template.

The reason I'm starting this discussion is because I didn't see this function regarding templates.

**Comment by vkbo** [comment]
Do you mean for the whole project, or individual documents?

**Reply by xahodo** [reply]
I mean for individual templates.

So, if I have a character template, I can export it under some name. Sharing my template with a friend would then be easier.

**Reply by vkbo** [reply]
Exporting single documents is coming. See #1001.

Templates are just regular documents. The only thing special about them is that they live in the "Templates" folder.
